### PR TITLE
Readme: Add setting JWT secret in compose file

### PR DESCRIPTION
**Pull request by adepssimius**
To be merged once the experimental image is latest. The container will not start without this env var set.

### CODE REVIEW

**Comment by pbogre** [comment]
Thanks for the PR.

I have indeed altered the README according to the new changes, but it is currently in my local repo waiting to be pushed. I will certainly add the suggestion to use the `openssl` command to generate one, but I'm closing the PR since the change has already been made

**Comment by adepssimius** [comment]
@pbogre I included the node command to generate a secret as well since I was confident it would be more cross platform. I know _I_ have openssl on my machine and it should be present on most linux boxes, but I can't remember if that was something I installed via homebrew at this point or what. I don't have any idea if that command works on a Windows machine. Granted not everybody has node, but at least it would be available in theory.

The other option was to call the crypto commands from a browser console which would definitely be cross platform, but I wouldn't want to do that given the access that extensions have in my browser.

Basically my intent was to give options to Windows users in a general way.

**Comment by pbogre** [comment]
@adepssimius I agree that it makes sense to provide a more cross-platform alternative to the command, and it is not a problem at all to add one more anyway so why not. I have added both commands in the [installation wiki](https://github.com/pbogre/jetlog/wiki/Installation#environment-variables)

Thanks again 🙂

PS: I have never installed openssl explicitly but the command is available to me on MacOS. It may be that it came as a dependency for something else though
